cmake/modules/RootMacros.cmake

```diff
@@ -299,6 +388,17 @@ function(ROOT_GENERATE_DICTIONARY dictionary)
     endforeach()
   endif()
 
+  set(headerdirs_dflt)
+
+  if(CMAKE_PROJECT_NAME STREQUAL ROOT)
+    if(EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/inc)
+       list(APPEND headerdirs_dflt ${CMAKE_CURRENT_SOURCE_DIR}/inc)
+    endif()
+    if(EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/v7/inc)
+       list(APPEND headerdirs_dflt ${CMAKE_CURRENT_SOURCE_DIR}/v7/inc)
```

Comment:
These default include paths are already collected some place else. Can we remove the duplication?

Reply:
It is old code, will be used now for non-ROOT dictionaries and ROOT tests.
I could try